niap-cc/Permissions/Tester/all-testers.json

```diff
@@ -0,0 +1,2524 @@
+{
+  "testCategories": [
+    {
+      "name": "internal",
+      "tests": [
+        {
+          "maxVersion": 2147483647,
+          "minVersion": 31,
+          "name": "android.permission.BYPASS_ROLE_QUALIFICATION"
+        },
+        {
+          "maxVersion": 2147483647,
+          "minVersion": 33,
+          "name": "android.permission.SET_DEFAULT_ACCOUNT_FOR_CONTACTS"
+        },
+        {
+          "maxVersion": 2147483647,
+          "minVersion": 33,
+          "name": "android.permission.SEND_SAFETY_CENTER_UPDATE"
+        },
+        {
+          "maxVersion": 2147483647,
+          "minVersion": 33,
+          "name": "android.permission.REQUEST_COMPANION_PROFILE_AUTOMOTIVE_PROJECTION"
+        },
+        {
+          "maxVersion": 2147483647,
+          "minVersion": 31,
+          "name": "android.permission.PERFORM_IMS_SINGLE_REGISTRATION"
+        },
+        {
+          "maxVersion": 32,
```

Comment:
Is it intentional that the max version is less than the min version?

In general, how is this file used? I didn't see it referenced anywhere else in the project (although I could have missed it).

Reply:
Will remove it. The file is generated by the MainActivity.writeAllTesterDetailsToJson().

I wrote this method for comparing  the differencies between actual tester implementation and manifest file.
If the max sdk version is less than min version , it shows the corresponding test is broken.

Reply:
Sounds good; thanks!